lib/utils/cli-flags.js

```diff
@@ -182,27 +192,31 @@ module.exports = {
         {
             name: 'target',
             alias: 't',
+            usage: '--target <build target>',
```

Comment:
Maybe we should show the targets available: https://webpack.js.org/configuration/target/#target